src/components/apps/launch/AppLaunchForm.js

```diff
@@ -588,90 +179,43 @@ const AppLaunchForm = (props) => {
 
     const hasParams = groups?.find((group) => group.parameters?.length > 0);
 
-    const hasReferenceGenomes =
-        hasParams &&
-        groups?.find((group) =>
-            group.parameters?.find((param) =>
-                ReferenceGenomeParamTypes.includes(param.type)
-            )
-        );
-
     const stepperRef = React.useRef(null);
     const [stepperHeight, setStepperRef] = useComponentHeight();
 
-    const bottomNavRef = React.useRef(null);
-    const [bottomNavHeight, setBottomNavRef] = useComponentHeight();
-
-    const appBarHeight =
-        stepperHeight +
-        appInfoHeight +
-        bottomNavHeight +
-        (isMobile ? MOBILE_NAVIGATION_BAR_HEIGHT : NAVIGATION_BAR_HEIGHT);
-
     React.useEffect(() => {
         setStepperRef(stepperRef);
     }, [stepperRef, setStepperRef]);
 
-    React.useEffect(() => {
-        setBottomNavRef(bottomNavRef);
-    }, [bottomNavRef, setBottomNavRef]);
-
-    React.useEffect(() => {
-        if (hasReferenceGenomes) {
-            setReferenceGenomesQueryEnabled(true);
-        }
-    }, [props.app, hasReferenceGenomes]);
-
-    const { isFetching: referenceGenomesLoading } = useQuery({
-        queryKey: [REFERENCE_GENOMES_QUERY_KEY, { deleted: false }],
-        queryFn: getReferenceGenomes,
-        config: {
-            enabled: referenceGenomesQueryEnabled,
-            onSuccess: (resp) => {
-                const genomes = resp?.genomes || [];
-                setReferenceGenomes(
-                    stableSort(genomes, (a, b) => a.name.localeCompare(b.name))
-                );
-                setReferenceGenomesQueryEnabled(false);
-            },
-            onError: (e) => {
-                console.error(e);
-            },
-        },
-    });
```

Comment:
I initially put this reference genomes listing query at the top-level of the App Launch form with the intention that the listing would only have to be fetched once, no matter how many reference genome selectors were included in the form.

It turns out that react-query is already smart enough to cache the listing for some amount of time after it's fetched once, at least on the server-side. For example, now when I run my example app in dev mode with this code, where 3 reference genome selectors are running their own query for the listings, I will see 3 requests from the browser to dev server, but I only see 1 request in the console from the server to the API.